internal/pkg/manifest/validate.go

```diff
@@ -1498,6 +1498,15 @@ func (s SidecarConfig) validate() error {
 	}
 	return s.ImageOverride.validate()
 }
+func (s SidecarConfig) validateSidecarImageFields() error {
```

Comment:
nit: the "sidecar" keyword in the function name is redundant with the struct name
```suggestion
func (s SidecarConfig) validateImages() error {
```